oefen_api/text-api.py

```diff
@@ -9,7 +9,7 @@ def schrijf_json():
 
     obj = json.loads(data)
     x = json.dumps(obj, sort_keys=False)
-    return str(x)
```

Comment:
Probeer je namen/termen/zinnen/comments/woorden allemaal in het engels te houden. `schrijf_json()` Probeer ook namen te geven aan je functies die iets beter beschrijven wat er in de functie gebeurt. B.v `def data_io_controller():`